lib/screens/home_page.dart

```diff
@@ -19,9 +23,13 @@ class HomePage extends StatefulWidget {
 }
 
 class _HomePageState extends State<HomePage> {
+  final GlobalKey<ScaffoldState> _scaffoldKey = GlobalKey<ScaffoldState>();
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
+      key: _scaffoldKey,
+      drawer: _buildAppDrawer(context),
       appBar: _buildAppBar(context),
```

Comment:
App Bar is different for both Admin and Customer so this has to be made different as well!